src/providers.ts

```diff
@@ -36,6 +37,7 @@ const LANGUAGE_EXTENSIONS: { [lang: string]: string } = {
   hpp: "cpp",
   lua: "lua",
   rs: "rust",
+  rb: "rubyb",
```

Comment:
Should this not be just `ruby`? https://code.visualstudio.com/docs/languages/identifiers

Reply:
It should, not sure where this b snuck in from